bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/SpanContextExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package com.bugsnag.android.performance.internal
+
+import com.bugsnag.android.performance.SpanContext
+
+/**
+ * Allows getting/setting the Span Context stack for the current thread.
+ *
+ * This is utilized by the BugSnag Performance Coroutines plugin and is intended for
+ * internal use only.
+ */
+@get:JvmSynthetic
+@set:JvmSynthetic
```

Comment:
I don't think we need to make these synthetic since it is already in the `internal` package.